src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
Rename `navigation__items-list__item` -> `navigation__item`. 

The BEM looks like: block, block__element, block__element--modifier. So we use `navigation__item` as `block__element`, not 'block__element__inner-element'

Reply:
Do it for the rest of html page